design-proposals/generated-disk-serial.md

```diff
@@ -0,0 +1,151 @@
+# Overview
+This proposal introduces a new feature in KubeVirt to automatically
+generate serial numbers for each disk attached to a virtual machine.
+
+## Motivation
+Some applications running in the VM require the disk serial number to be defined and persistent across reboots.
+Generating the serial reduces manual configuration, and in the future it can be part of a `VirtualMachinePreference` for the OS. 
+
```

Comment:
Is there a reference (doc or implemention) how this is addressed in other virtualization management system, especially oVirt and OpenStack?

Reply:
I will check oVirt code.
@lyarwood , do you know how this is done in OpenStack? Or can you point me to docs or source, where I could find it?

Reply:
Thanks!

oVirt uses the disk GUID as a serial number. It would be similar if we used the UID of a PVC.